### PR TITLE
fix: Adjust percentage indicator color

### PR DESCRIPTION
**Pull request by nbetsaif**
This PR adjusts the color of the percentage indicator

## Screenshots
Before            |  After
:-------------------------:|:-------------------------:
before  |  after

### CODE REVIEW

File: lib/core/extension/proposal_model_extensions.dart
Comment (n13) [inline lib/core/extension/proposal_model_extensions.dart:35]: it would be nice to have an "isPassing" function for this on proposal model

IsPassing should be interesting not just for color, but also for other purposes. 

Also, the quorum percentage numbers are settings on the DAO, they can't be hard-coded. 

They need to be loaded from the DAO

Since this is a UX change, I will approve and leave it up to you if you want to fix one or both of these items in this PR or make a todo and fix it in another PR.

**Review by n13 — APPROVED** [review]
Approved with comments - needs a fix or another PR to work correctly.

File: lib/core/extension/proposal_model_extensions.dart
Comment (nbetsaif) [reply lib/core/extension/proposal_model_extensions.dart:35]: Thank you for the feedback @n13 ! Regarding the quorum percentage numbers, in our code, we don't hard-code them. We actually receive the percentages for both quorum and unity from the GraphQL API. The only adjustment we make is reformatting them from int to double, as the package we use for displaying percentages requires double as input, not int.

As for the isPassing function, I agree that it would be a good idea to implement it for broader use cases beyond just color. I'll handle it in this PR

File: lib/core/extension/proposal_model_extensions.dart
Comment (n13) [inline lib/core/extension/proposal_model_extensions.dart:33]: the values .2 and .8 are still hard-coded here?

File: lib/core/extension/proposal_model_extensions.dart
Comment (n13) [reply lib/core/extension/proposal_model_extensions.dart]: There's 2 values in here, 0.2 and 0.8, which are hard-coded. 

Not sure if you're referring to other places in the code where these values are loaded from the server, but they need to be loaded from the server also here, when we indicate fail/pass state ;)

File: lib/core/extension/proposal_model_extensions.dart
Comment (nbetsaif) [reply lib/core/extension/proposal_model_extensions.dart:35]: Yes, the values .2 and .8 are still hard-coded for now. I've added a TODO in the code to update these values later by fetching them dynamically from the server, as they are relative to each DAO.

**Pull request merged**
